.env.example

```diff
@@ -0,0 +1,7 @@
+  SECRET_KEY=supersecretkey
+  DEBUG=True
+  DB_NAME=yourdbname
+  DB_USER=yourname
+  DB_PASSWORD=yourstrongpassword
+  DB_HOST=127.0.0.1
+  
```

Comment:
Also, take out the .env.example file and add that to the readme.

Reply:
Okay @abulojoshua1, although the point of us having the `.env.example` file is so that once you clone the repo you simply rename the file to .env and then customize the provided fields. Currently, you have to create a .env every time you clone the repo and we found it quite tedious. I will remove the file because it is already on the README

Reply:
@MandelaK, Your point makes sense but it is done just once and not repeated. I do not think you guys will have to constantly clone the repo.

Reply:
@abulojoshua1, If I can jump in, some of us are doing reviews on a freshly cloned repo each time, to avoid polluting our unfinished work. If you can allow it, I feel it will make reviewing PRs much easier and stress-free.